backend/spec/requests/tags_spec.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Tags' do
+  let!(:medium_priority_tag) { create(:tag, priority: 2) }
+  let!(:high_priority_tag) { create(:tag, priority: 1) }
+  let!(:low_priority_tag) { create(:tag, priority: 3) }
+  let(:tags) { [high_priority_tag, medium_priority_tag, low_priority_tag] }
```

Comment:
Idiomatic/Simple

かなり変数を定義していて、冗長なので、以下のようにするのはいかがでしょうか？
（tagsはtags.valuesで取得できますし、各keyにアクセスすることで、それぞれのpriorityのTagが取得できます。）

```suggestion
  # Example result:
  # {
  #   1 => tag with priority 1,
  #   2 => tag with priority 2,
  #   3 => tag with priority 3
  # }
  let!(:tags) do
    (1..3).map do |index|
      create(:tag, priority: index)
    end.index_by(&:priority)
  end
```

Reply:
上記の書き方だとかなりスッキリしますね！
提案いただいた内容で修正したいと思います🙏